Periodic_4_hyperbolic_triangulation_2/test/Periodic_4_hyperbolic_triangulation_2/test_p4ht2_remove_dummy_points.cpp

```diff
@@ -0,0 +1,94 @@
+#include <CGAL/CORE_Expr.h>
+#include <CGAL/Cartesian.h>
+
+#include <CGAL/point_generators_2.h>
+//#include <CGAL/Random_points_in_disc_2.h>
```

Comment:
Can be cleaned from the multiple places it appears in.

Reply:
I've cleaned the code of the tests to remove unneeded comments.